Buildplan/Dockerfile-Jenkins

```
@@ -67,8 +67,10 @@ ENV COMPOSER_H_DIR="${HOME}/.composer"
 ENV PLUGIN_DIR="${WP_PLUGINS_DIR}/smartling-connector"
 ENV LOCAL_GIT_DIR="/plugin-dir"
 
-COPY Buildplan/build.sh /build.sh
-RUN chmod +x /build.sh
+COPY Buildplan/release.sh /release.sh
+COPY Buildplan/test.sh /test.sh
+RUN chmod +x /release.sh
+RUN chmod +x /test.sh
```

JFYI, can be single line
```suggestion
RUN chmod +x /release.sh /test.sh
```